recipe/meta.yaml

```diff
@@ -8,12 +8,15 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/robyn-{{ version }}.tar.gz
-  sha256: b968bc3a1c9c7093c71998de36312d207fdddd0762b0f669204f0d5aa08e4aa8
+  sha256: 5e6ee3b8b047f4707f10200a75cbf15e5c1d8b981a22cc71abe2d5111c642f98
 
 build:
   skip: true  # [py<37]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script:
+    # TODO remove this when rust-activation-feedstock is fixed
+    - export CARGO_BUILD_RUSTFLAGS="$CARGO_BUILD_RUSTFLAGS -C link-arg=-Wl,-headerpad_max_install_names -C link-arg=-Wl,-dead_strip_dylibs"  # [osx]
```

Comment:
Can you send a PR to `rust-activation-feedstock` with this change to activate.sh?

Reply:
I'll try

Reply:
@isuruf with the OSX selector or without?

Reply:
@isuruf oh or maybe I do it under:
```
elif [[ "@cross_target_platform@" == osx* ]]; then
```
Would that work?

Reply:
@isuruf here's the PR

Reply:
https://github.com/conda-forge/rust-activation-feedstock/pull/21